metron-analytics/metron-maas-common/src/main/java/org/apache/metron/maas/discovery/ServiceDiscoverer.java

```diff
@@ -329,7 +324,7 @@ public ModelEndpoint getEndpoint(Model model) {
    * List all endpoints for a given model.
    *
    * @param model
-   * @return
+   * @return ret
```

Comment:
Can you make this more descriptive? e.g. Mapping of models to a respective list of endpoints.